### DISCUSSION TITLE
Nuxt 3 and Tina setup

### DISCUSSION THREAD

**Discussion by cnaallotey** [tinacms/tinacms#4401] [Help]
Encountering an error when starting my dev server. I am running on Nuxt 3 . Any assistance?
![Screenshot 2023-12-21 at 2 32 51 PM](https://github.com/tinacms/tinacms/assets/69573740/9c9e5a4c-2a61-4505-adf7-d18be05ebd85)

**Comment by kldavis4** [comment]
This seems to be a dependency issue: https://discord.com/channels/835168149439643678/1186446149930983454/1186446149930983454 

Join the tina discord here: https://discord.com/invite/zumN63Ybpf

**Comment by kldavis4** [comment]
Also see https://github.com/tinacms/tinacms/issues/4400
